cli/dcoscli/data/help/node.txt

```diff
@@ -13,6 +13,7 @@ Usage:
     dcos node list-components [--leader --mesos-id=<mesos-id> --json]
     dcos node log [--follow --lines=N --leader --master --mesos-id=<mesos-id> --slave=<agent-id>]
                   [--component=<component-name> --filter=<filter>...]
+    dcos node metrics (--mesos-id=<mesos-id>) [--filter=<name>... | --json]
```

Comment:
this is required, right? in that case we should do `dcos node metrics <mesos-id> [....]`